source/acquaman/AMContinuousScanActionController.cpp

```diff
@@ -725,11 +777,12 @@ bool AMContinuousScanActionController::placeInterpolatedDataInDataStore()
 
 bool AMContinuousScanActionController::cleanupClientDataRequests()
 {
-	QMap<QString, AMDSClientDataRequest*>::const_iterator j = clientDataRequestMap_.constBegin();
-	while(j != clientDataRequestMap_.constEnd()){
-		j.value()->deleteLater();
-		j++;
-	}
+//	QMap<QString, AMDSClientDataRequest*>::const_iterator j = clientDataRequestMap_.constBegin();
```

Comment:
Clearing rather than deleting?  Isn't this a memory leak or is it dealt with somewhere else?